src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -165,6 +172,15 @@ public TopicPartitionWriter(
       this.hiveUpdateFutures = hiveUpdateFutures;
       hivePartitions = new HashSet<>();
     }
+
+    if(rotateScheduleIntervalMs > 0) {
+      String timeZoneString = (String) connectorConfig.getString(HdfsSinkConnectorConfig.TIMEZONE_CONFIG);
+      if (timeZoneString.equals("")) {
+        throw new ConfigException(HdfsSinkConnectorConfig.TIMEZONE_CONFIG,
+                timeZoneString, "Timezone cannot be empty.");
```

Comment:
The error message should be more explicit. How about "timezone cannot be empty when using scheduled file rotation."